SConstruct

```diff
@@ -317,8 +317,8 @@ def check_for_release_target():  # pylint: disable=too-many-locals
         try:
             remote.push(['refs/heads/{}'.format(branch)],
                         callbacks=MyCallbacks())
-        except pygit2.GitError as excpt:
-            print("Error pushing branch: {}".format(excpt))
+        except pygit2.GitError as err:
```

Comment:
I'm surprised that codespell is checking these, as "codespell" it's supposed to check comments but not code.  Still, no bother and this is better.